virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -11,6 +11,12 @@ toc::[]
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
 
+[id="virt-4-14-doc-feedback"]
+== Providing feedback on {VirtProductName} documentation
+
+If you have a suggestion for improving this documentation or have found an error, submit a link:https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12323181&issuetype=1&components=12333768&priority=10200&summary=%5BDoc%5D&customfield_12316142[Jira issue]. Please provide a brief summary of the issue and specific details, such as the section name and {VirtProductName} version.
```

Comment:
I see that this is the same text that OCP uses but it is odd. We do not normally use "Please" in our docs. It's also much more verbose than our usual doc style. I want to see whether we can make the form more user-friendly (boilerplate text is a bit much for docs) but I do not want to hold up your PR. 

I just noticed a chicken/egg problem. The user has to be logged in to use the form, which is not an ideal workflow.

Maybe this: 

"To report an error or to iimprove our documentation, log into your Red Hat Jira account(link:https://issues.redhat.com) and submit a Jira issue(form)."

I'm not sure it's necessary to tell them what to fill in because the boilerplate text tells them. (Most users will probably ignore it anyway.)